PhotoCollection/PhotoCollection/View Controllers/ThemeSelectionViewController.swift

```diff
@@ -9,16 +9,56 @@
 import UIKit
 
 class ThemeSelectionViewController: UIViewController {
-
-    func selectDarkTheme() {
+    
+    var themeHelper: ThemeHelper?
+    
+    @objc func selectDarkTheme() {
         themeHelper?.setThemePreferenceToDark()
         dismiss(animated: true, completion: nil)
     }
     
-    func selectBlueTheme() {
+    @objc func selectBlueTheme() {
         themeHelper?.setThemePreferenceToBlue()
         dismiss(animated: true, completion: nil)
     }
     
-    var themeHelper: ThemeHelper?
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setUpSubviews()
+    }
+    
+    private func setUpSubviews() {
+        let label = UILabel()
+        label.text = "Select theme:"
+        label.font = .boldSystemFont(ofSize: 30)
+        label.textAlignment = .center
+        view.addSubview(label)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            label.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 60),
+            label.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: 20),
+            label.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -20),
+        ])
+        
```

Comment:
Your code looks great! Keep up the consistent formatting.

Reply:
Thanks!